### ISSUE TITLE
Similar to #1200 Ledger live 2.0.0 update - The password you entered is incorrect

### ISSUE THREAD

**Issue by Kjax26** [LedgerHQ/ledger-live-desktop#2692]
- tested on Ledger Live 2.0.0 (just downloaded)
- Platform and version: Win10 64Bit

#### Expected behavior

Use my password that I have used since always.

#### Actual behavior

Updated Ledger nano S to v1.6 successfully and also ran update on Ledger Live (desktop) to 2.0.0. It no longer wants to let me sign in. 

Is choosing the option "I lost my password" the only way to sort this?

**Comment by Kjax26** [comment]
Downgraded back to v1.1. Able to login...

**Comment by gre** [comment]
@Kjax26 if you try 2.0 and then later downgrade to a previous version, this is expected. the encryption algorithm has changed and a migration happened when you upgraded to 2.0. the accounts data will not be able to be loaded in a 1.x version of Ledger Live. but if you go back to 2.0 it should work.

If it doesn't case, you can try to delete/move the Ledger Live folder under the "user data folder".


**Location of User Data Folder (copy in the file address bar):**

- on MacOS: `$HOME/Library/Application\ Support/Ledger\ Live`
- on Linux: `$HOME/.config/Ledger\ Live`
- on Windows: `%APPDATA%\Roaming\Ledger Live`

(or alternatively, if you see hidden files: C:\Users\<username>\AppData\Roaming\Ledger Live)

**Comment by Kjax26** [comment]
@gre  The problem was moving forward to 2.0, saying my password is incorrect but knowing it was correct.

The fix was what you described thank you!


***FIX for anyone should they be having the same issues:***

WINDOWS FIX

Go to browser and find the hidden files: C:\Users<username>\AppData\Roaming\Ledger Live)

(update with your computers username obviously)

I moved the Ledger Live folder in this instance to retain what information I had as precaution. Then clean installed a new version of v2.0. 

Upon setup, I chose the option to install without recovery phrase (24 words). Linked my Ledger Nano S and entered the pin... DONE!


Thank you!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
